scripts/release_scripts/wrap_up_release_test.py

```diff
@@ -27,9 +27,7 @@
 from scripts import common
 from scripts.release_scripts import wrap_up_release
 
-# pylint: disable=wrong-import-position
-import github  # isort:skip
-# pylint: enable=wrong-import-position
+import github  # isort:skip  pylint: disable=wrong-import-position
```

Comment:
Can you please check this one as well? (as above)

Reply:
Fixed.